frontend/components/branchViewLink.js

```diff
@@ -7,32 +7,33 @@ import { IconView } from './icons';
 const isDefaultBranch = (branchName, site) => branchName === site.defaultBranch;
 const isDemoBranch = (branchName, site) => branchName === site.demoBranch;
 
-const getUrlAndViewText = (branchName, site, completedAt) => {
-  if (isDefaultBranch(branchName, site)) {
-    return { url: site.viewLink, viewText: 'View site' };
-  }
-  if (isDemoBranch(branchName, site)) {
-    return { url: site.demoViewLink, viewText: 'View demo' };
-  }
+const getUrlAndViewText = (branchName, buildURL, site, completedAt) => {
+  const urlWithSlash = (rawUrl) => {
```

Comment:
Instead of duplicating this logic in the frontend, can we just add the viewLink when serializing the build?